pkg/operator/ceph/cluster/cleanup.go

```diff
@@ -201,6 +203,10 @@ func (c *ClusterController) getCephHosts(namespace string) ([]string, error) {
 			return hostNameList, errors.Wrapf(err, "could not list the %q pods", app)
 		}
 		for _, cephPod := range podList.Items {
+			if cephPod.Status.Phase != v1.PodRunning {
+				logger.Debugf("ignoring daemon %q with phase %q", cephPod.Name, cephPod.Status.Phase)
```

Comment:
An info message would be preferred in this case so we can see why it wasn't cleaned up
```suggestion
				logger.Infof("ignoring daemon %q with phase %q", cephPod.Name, cephPod.Status.Phase)
```